lib/device-log/ios-log.js

```diff
@@ -4,8 +4,8 @@ import logger from './logger';
 import { fs, mkdirp } from 'appium-support';
 import xcode from 'appium-xcode';
 import { SubProcess } from 'teen_process';
-import getHeapStatistics from 'v8';
 
+const v8 = require('v8');
```

Comment:
Can't this just be `import v8 from 'v8';`? Seems to work locally form me, and doesn't require a different style of importing.